RecoTracker/MkFit/plugins/createPhase1TrackerGeometry.cc

```diff
@@ -0,0 +1,372 @@
+    setupIterationParams(ii[0].m_params, 0);
+    ii[0].m_partition_seeds = PartitionSeeds0;
+    fillHitSelectionWindowsParams(ii[0]);
+
+    ii[1].Clone(ii[0]);  //added extra iterations with some preliminary setup
+    setupIterationParams(ii[1].m_params, 1);
+    ii[1].set_iteration_index_and_track_algorithm(1, (int)TrackBase::TrackAlgorithm::highPtTripletStep);
+    ii[1].set_seed_cleaning_params(2.0, 0.018, 0.018, 0.018, 0.018, 0.018, 0.05, 0.018, 0.05);
+    fillHitSelectionWindowsParams(ii[1]);
+
+    ii[2].Clone(ii[0]);
+    setupIterationParams(ii[2].m_params, 2);
+    ii[2].set_iteration_index_and_track_algorithm(2, (int)TrackBase::TrackAlgorithm::lowPtQuadStep);
+    ii[2].set_seed_cleaning_params(0.5, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05);
+    fillHitSelectionWindowsParams(ii[2]);
+
+    ii[3].Clone(ii[0]);
+    setupIterationParams(ii[3].m_params, 3);
+    ii[3].set_iteration_index_and_track_algorithm(3, (int)TrackBase::TrackAlgorithm::lowPtTripletStep);
+    ii[3].set_seed_cleaning_params(0.5, 0.0, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05);
+    fillHitSelectionWindowsParams(ii[3]);
+
+    ii[4].Clone(ii[0]);
+    setupIterationParams(ii[4].m_params, 4);
+    ii[4].set_iteration_index_and_track_algorithm(4, (int)TrackBase::TrackAlgorithm::detachedQuadStep);
+    ii[4].set_seed_cleaning_params(2.0, 0.018, 0.018, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05);
+    fillHitSelectionWindowsParams(ii[4]);
+
+    ii[5].Clone(ii[0]);
+    setupIterationParams(ii[5].m_params, 5);
+    ii[5].set_iteration_index_and_track_algorithm(5, (int)TrackBase::TrackAlgorithm::detachedTripletStep);
+    ii[5].set_seed_cleaning_params(2.0, 0.018, 0.018, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05);
+    fillHitSelectionWindowsParams(ii[5]);
+
+    ii[6].Clone(ii[0]);
+    setupIterationParams(ii[6].m_params, 6);
+    ii[6].set_iteration_index_and_track_algorithm(6, (int)TrackBase::TrackAlgorithm::mixedTripletStep);
+    ii[6].set_seed_cleaning_params(2.0, 0.05, 0.05, 0.135, 0.135, 0.05, 0.05, 0.135, 0.135);
+    fillHitSelectionWindowsParams(ii[6]);
+
+    ii[7].Clone(ii[0]);
+    setupIterationParams(ii[7].m_params, 7);
+    ii[7].set_iteration_index_and_track_algorithm(7, (int)TrackBase::TrackAlgorithm::pixelLessStep);
+    ii[7].set_seed_cleaning_params(2.0, 0.135, 0.135, 0.135, 0.135, 0.135, 0.135, 0.135, 0.135);
+    ii[7].set_qf_flags();
+    ii[7].set_qf_params(4, 0.19);
+    fillHitSelectionWindowsParams(ii[7]);
+
+    ii[8].Clone(ii[0]);
+    setupIterationParams(ii[8].m_params, 8);
+    ii[8].set_iteration_index_and_track_algorithm(8, (int)TrackBase::TrackAlgorithm::tobTecStep);
+    ii[8].set_seed_cleaning_params(2.0, 0.135, 0.135, 0.135, 0.135, 0.135, 0.135, 0.135, 0.135);
+    ii[8].set_qf_flags();
+    ii[8].set_qf_params(4, 0.25);
+    fillHitSelectionWindowsParams(ii[8]);
+
+    //for the latter 2 iter investing in maxCand & stop condition (for time) + QF and Dupl. cleaning (for quality)
+
+    if (verbose) {
```

Comment:
I agree, but because of needing changes in the mkFit external I'd prefer to leave that to later time when the external is updated for the next time. I'll add a TODO comment in the code.